tests/unit/tabular/test_base.py

```diff
@@ -61,7 +61,7 @@ def test___init__metadata_object(self):
 
         # Run
         gc = GaussianCopula(
-            default_distribution='bounded',
+            default_distribution='gamma',
```

Comment:
how were the new default distributions for these tests picked?

Reply:
I just randomly chose them, since they are pretty much interchangable.